package.json

```diff
@@ -18,7 +18,7 @@
     "postbuild": "dts-bundle-generator src/types/index.ts --out-file lib/index.d.ts",
     "mkdirs": "mkdir -p dist && mkdir -p lib",
     "clean": "rimraf lib dist",
-    "prepare": "npm run clean && npm run check && npm run build",
+    "prepare": "npm run clean",
```

Comment:
Is this a permanent change or just temporarily during development? :thinking:

Reply:
Temporary. Brought it back